docs/resources/logdna_index_rate_alert.md

```diff
@@ -0,0 +1,73 @@
+# Resource: `logdna_index_rate_alert`
+
+Manages [LogDNA Index Rate Alert](https://docs.mezmo.com/docs/index-rate-alerts). Configuring alerts based on the index rate or retention and storage rate of your log data helps you track unusual behavior in your systems. For example, if there's a sudden spike in volume,. Mezmo's Index Rate Alert feature tells you which applications or sources produced the data spike. It also shows any recently added sources. Index rate alerts can also help managers who are responsible for budgets to analyze and predict storage costs.
+
+To get started, all you need to do is to specify a configuration and one of our currently supported alerts recipients: email, Slack, or PagerDuty.
+
+Be aware, the only one index rate alert configuration is allowed per account
+
+## Example - Index Rate Alert
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) specify a LogDNA region
+}
+
+resource "logdna_index_rate_alert" "config" {
+  max_lines = 3
+  max_z_score = 3
+  threshold_alert = "separate"
+  frequency = "hourly"
+  enabled = true
+  channels {
+    email = ["test@email.test"]
```

Comment:
test@email.test in the example if used by user to try out, it will result a 400 response telling users to use a valid email address. Maybe having .com email address will get the user a first 200 response.